guide/inheritance-wallet/succession.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: guide
+title: Succession & fund transfer
+description: This page describes how Christina and Davide recover the wallet by using their inheritance keys.
+nav_order: 6
+parent: Inheritance wallet
+permalink: /guide/inheritance-wallet/succession/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/succession/preview.png
+image_base: /assets/images/guide/inheritance-wallet/succession/
+images_wallet-import:
+    - file: wallet-import/app-home-initial
+      alt: Screen showing an empty app home screen.
+      caption: Christina taps the plus button to start adding the wallet.
+    - file: wallet-import/add-wallet-sheet
+      alt: Screen showing different options for adding a wallet.
+      caption: Christina selects "Import wallet"
+    - file: wallet-import/christina-import-config-options
+      alt: Screen showing different options for importing the wallet configuration.
+      caption: She chooses to scan the QR code from the recovery PDF. 
+    - file: wallet-import/christina-scan-config
+      alt: Screen showing an open QR code scanner.
+      caption: Christina scans the QR code.
+    - file: wallet-import/christina-review-wallet
+      alt: Screen showing the final wallet configuration, with an option to review the details of the signing keys.
+      caption: She reviews the wallet configuration and finishes the import.
+    - file: wallet-import/christina-wallet-success
+      alt: Screen showing a success message and the option to download a wallet backup kit. 
+      caption: Christina skips the download of the backup kit, as she already has it.
+    - file: wallet-import/app-home-done
+      alt: App home screen showing the family savings wallet.
+      caption: The home screen reminds Christina that she should download the backup kit and enable her signing device.
```

Comment:
Why would the user need to download another backup kit? Wouldn’t the same backup kit they just used to restore the wallet still be valid? Iirc unless the user is moving the funds to another multisig wallet set up with a new quorum, the same recovery output descriptors should always remain valid.

 If this is already explained in the previous sections, then just let me know and please disregard this comment. No need to explain or tell me how this works in the comments. Just flagging.

Reply:
That's a very good point. She already has the file, plus it's available in the wallet settings section. The only thing left for her to do is to activate her signing device. Will update the mockups accordingly tomorrow.